### PR TITLE
Spanish (Spain) translation

### PR DESCRIPTION
**Pull request by treborin**

### CODE REVIEW

**Comment by datamweb** [comment]
Hey @treborin ,
Thank you for contributing!
Please see https://github.com/codeigniter4/shield/pull/235#issuecomment-1153050805

**Comment by treborin** [comment]
> Hey @treborin , Thank you for contributing! Please see [#235 (comment)](https://github.com/codeigniter4/shield/pull/235#issuecomment-1153050805)

@datamweb . Done!

File: tests/Language/AbstractTranslationTestCase.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ abstract class AbstractTranslationTestCase extends TestCase
         //        BosnianTranslationTest::class            => 'bs',
         //        CzechTranslationTest::class              => 'cs',
         GermanTranslationTest::class => 'de',
-        //        SpanishTranslationTest::class            => 'es',
+        SpanishTranslationTest::class                      => 'es',
```
Comment (datamweb) [inline tests/Language/AbstractTranslationTestCase.php:52]: Please fix the coding style. **just run :**
`composer cs-fix`

**Review by datamweb — CHANGES_REQUESTED** [review]

**Comment by datamweb** [comment]
It looks like you are using GitHub UI to post this PR.
Please rename file `tests/Language/SpanishTranslationTest` to `tests/Language/SpanishTranslationTest.php`.
Then tell me if there is any problem in executing command `composer cs-fix`?

**Comment by datamweb** [comment]
> image

@kenjis , Does executing command `composer update` solve this problem?

**Comment by kenjis** [comment]
@treborin run `composer update` and `composer cs-fix` in the repository root directory, that is `shield`.

**Comment by treborin** [comment]
> @treborin run `composer update` and `composer cs-fix` in the repository root directory, that is `shield`.

@kenjis , cmposer update, ok. But here is composer cs-fix:
image

@datamweb, i fixed coding style by hand. I saw inside unsuccessfuls checks thay te problem was the file text format, that need to be ansii. And i fixed it by hand, line by line.

**Comment by datamweb** [comment]
Yes, I got it.
try 
`mkdir build`
`composer update` 
`composer cs-fix`

**Comment by treborin** [comment]
> Yes, I got it. try `mkdir build` `composer update` `composer cs-fix`

image
image

**Comment by MGatner** [comment]
The `cs-fix` command was just added this week (#340). If your fork was from earlier the command would be `composer style`. Either way that is simply an alias, you can run the command directly:
```shell
php vendor/bin/php-cs-fixer fix --ansi
```

**Comment by MGatner** [comment]
I don't know what is happening with those action failures. The tests all pass but a PHP error occurs when it tries to gather coverage. I believe it is unrelated to this PR.

**Comment by treborin** [comment]
> I don't know what is happening with those action failures. The tests all pass but a PHP error occurs when it tries to gather coverage. I believe it is unrelated to this PR.

I try with a new PR

**Pull request closed without merging**